src/shared/libfido2-util.c

```diff
@@ -450,6 +450,19 @@ static int fido2_use_hmac_hash_specific_token(
         return 0;
 }
 
+static const char *fido2_algorithm_to_string(int alg) {
+        switch(alg) {
+                case COSE_ES256:
+                        return "ES256";
+                case COSE_RS256:
+                        return "RS256";
+                case COSE_EDDSA:
+                        return "EDDSA";
+                default:
+                        return NULL;
+        }
+}
```

Comment:
so almost all our string enumerations use lowercase spelling. I think we should continue that rule here, and thus this should be "es256", "rs256" and "eddsa"...

Reply:
Should I change the argument for the option too?

Reply:
the C enum/defines should be all uppercase, but the string representaiton we parse or output should be lowercase. That's how we do it for almost all enums or enum-like things.

but not sure i grok the question fully?

Reply:
So I think it's not necessary to make the argument(like ES256) of the newly added option(fido2-credential-algorithm) to lowercase.

Reply:
hmm? my point was to make all our config files/cmdline switches to accept the lowercase spelling only, as that's kinda what we do across thre tree for all our enum and enum-like settings

Reply:
Ok, I'll update the code.